embedding/cpp/elm_flutter_view.cc

```diff
@@ -16,8 +16,8 @@ bool ElmFlutterView::RunEngine() {
   // Read engine arguments passed from the tool.
   Utils::ParseEngineArgs(&engine_args_);
 
-  std::vector<const char *> switches;
-  for (auto &arg : engine_args_) {
+  std::vector<const char*> switches;
+  for (auto& arg : engine_args_) {
```

Comment:
Do you know what causes these style changes? Obviously the config file (`.clang-format`) hasn't changed.

Reply:
oh... 
All code has been changed according to the code format in the lambda function.
If I write the code below and format it, all symbols will be left-aligned.
```cpp
  evas_object_event_callback_add(
      evas_object_, EVAS_CALLBACK_DEL,
      [](void* data, Evas* evas, Evas_Object* object, void* event_info) {
        auto* self = reinterpret_cast<ElmFlutterView *>(data);
        if (self->engine_) {
          FlutterDesktopEngineShutdown(self->engine_);
          self->engine_ = nullptr;
        }
      },
      this);
```
However, if I write `auto *self ` , only the format of the argument is changed as we expected.